src/06-sort.js

```diff
@@ -23,7 +28,13 @@ function sortByRuntimeAscending(songs) {}
  * @param {Object[]} songs - An array of songs. See the song data for more.
  * @returns {Object[]}
  */
-function sortByArtistNameDescending(songs) {}
+function sortByArtistNameDescending(songs) {
+ const sortedNames = songs.sort((song1, song2)=> {
+  return song2.artist.toUpperCase() > song1.artist.toUpperCase();
+ }) 
+ return sortedNames;
```

Comment:
![Screenshot 2022-11-22 at 11 10 55 AM](https://user-images.githubusercontent.com/71154265/203364476-e8bf8d82-76bd-41ac-9cb8-10d8754f3edb.png)
how can you apply this logic to this problem?? Use conditionals too return 1, -1, 0r 0.